src/javascript/ContentEditor/DesignSystem/OrderableValue/OrderableValue.jsx

```diff
@@ -31,11 +37,11 @@ export const OrderableValue = ({field, onFieldRemove, onValueReorder, index, com
         >
             <div className={`${styles.referenceDropGhostHidden} ${isDropping ? styles.referenceDropGhost : ''}`} data-droppable-zone={name}/>
             {component &&
-                <div className={styles.draggableCard} {...(isReferenceCard && {ref: drag})}>
+                <div ref={isReferenceCard && isDraggable ? drag : null} className={styles.draggableCard}>
```

Comment:
`react-dnd` adds an attribute `draggable` that does not reflect whether an element can indeed be dragged or not from `canDrag` so it's not best to depend on react-dnd here. So we add drag ref only if we know element is draggable or not.

https://github.com/react-dnd/react-dnd/issues/2690